src/projective-skybox.ts

```diff
@@ -43,9 +48,10 @@ void main(void) {
     vec3 view_dir = normalize(vViewDir);
 
     // intersect ray with world geometry
-    float t = 100.0;
-    if (view_dir.y < 0.0) intersectPlane(t, view_pos, view_dir, groundPlane);
-    intersectSphere(t, view_pos, view_dir, domeParams);
+    float t = 8000.0;
```

Comment:
What's t? Why 8000?

Reply:
Maximum ray length

Reply:
I've added a comment to explain as much.